src/pudl/transform/ferc1.py

```diff
@@ -4003,6 +4044,189 @@ class UtilityPlantSummaryFerc1TableTransformer(Ferc1AbstractTableTransformer):
     table_id: TableIdFerc1 = TableIdFerc1.UTILITY_PLANT_SUMMARY_FERC1
     has_unique_record_ids: bool = False
 
+    def transform_main(self: Self, df: pd.DataFrame) -> pd.DataFrame:
+        """Spot fix depreciation_utility_plant_in_service records with bad signs."""
+        df = super().transform_main(df)
+
+        primary_keys = [
+            "report_year",
+            "utility_id_ferc1",
+            "utility_type",
+            "utility_plant_asset_type",
+        ]
+
+        # We need to track which years are getting transformed so the spot fixes
+        # don't fail on the fast tests.
+        df_years = df.report_year.unique().tolist()
+
+        # The utility_id_ferc1 211 follows the same pattern for several years
+        # instead of writing them all out in spot_fix_pks, we'll create a loop that
+        # generates all of them and then append them to spot_fix_pks later
+        spot_fix_211 = []
+        for year in np.append(2006, range(2009, 2021)):
+            # Measure to account for fast tests where not all years are used
+            if year in df_years:
```

Comment:
Is there a reason not to do this using the spot_fixer(), rather than inside of `transform_main()`?

Reply:
It makes more sense to keep them like this because there are over one hundred of these spot fixes and it would be very annoying to type all of those out in the current spot fixer (which doesn't accomadate any programatic creation of spot fixes)